src/work-allocation/containers/task-list/task-list.component.spec.ts

```diff
@@ -431,7 +431,16 @@ describe('TaskListComponent', () => {
     });
 
     it('should correctly set the summary text', () => {
-      expect(paginationSummary.textContent).toContain('Displaying 2 of 100 tasks');
+      wrapper.pagination = {
+        page_number: 1,
+        page_size: 10
+      };
+
+      wrapper.tasks = getTasks();
+
+      fixture.detectChanges();
+
+      expect(paginationSummary.textContent).toContain('Displaying 1 - 2 out of 100 tasks');
```

Comment:
Actually, just revisited this unit test... that doesn't make sense. If the page size is 10 and there are 100 tasks, it can't be showing "1-2". If there are only 2 tasks, it shouldn't be saying "out of 100 tasks".

Reply:
changed the test